src/scss/_breadcrumb.scss

```diff
@@ -43,6 +43,13 @@
   margin-right: px-to-rem(3);
 }
 
+.jazz-breadcrumb > ol > li > div {
```

Comment:
Is the "div" necessary here?  It look like you might be trying to use it get get some alignment.  Can't you achieve the same thing by putting these styles on the parent "li" element (.jazz-breadcrumb > ol > li) or on all if the li children (.jazz-breakcrumb > ol > li > i, .jazz-breakcrumb > ol > li > i ... or even .jazz-breakcrumb > ol > li > *)?

If the "div" is required, it is not a problem, but as much as possible we will want to eliminate the need to introduce divs to the structure just for the sake of formatting.

Reply:
That is definitely how I tried this first but unfortunately moving the styles to the li broke it and caused the items to display in separate lines.  I actually started from the ul level but ultimately ended up creating a div to align just that part of the dom.

Reply:
I don't think you want the style directly on the "li", but on the children of the li (li > *).  You are correct that putting the styles on the "li" would change the flex behavior of the "li" within the list.  You want to change the flex behavior of the children within the "li".

Reply:
Just tried this now and it applied the style to each item in the li which is not what we want as we would still need a div to align them.

Reply:
That probably means that you need to split the styles between the "li" and the "li > *".  You are adding a "div" element to use as a container that you can apply styles to, but the "li" is already a container that you can apply styles to.  If for some reason the "li" has some conflicting behavior, it might also be possible to stretch the flex behavior from the "ul" down to the "li > *", but I have never tried that.

Reply:
The styles are currently split up :) li > * means apply the style to any item in the li so it actually applied the flex style to each item (icon, text etc) but what I needed it to do is apply the style to something that contains it which in this case is the li.  When I did try to change the style of the li from inline-block it broke the alignment for the list item. I can try as you said to change the style in the ul but that didn't work last time I tried it. Let me do more research on this to see if there is some other way of doing it. I did try various things last week and this seemed to be the best way to do it.

Reply:
It could be there is no other way to do it, but I am always going to push back on the addition of divs.  There will be cases where we have no other choice, but this doesn't seem like one of them.  If the icons do not align properly in this case, then they would also not align properly in the case of a simple paragraph...and we would not be adding a div inside of a paragraph to align icons.  Do some more research and see if there are other options.  Here is a post that I came across that is using the same HTML structure that we have (although not for breadcrumbs...the idea should be the same): https://css-tricks.com/tips-aligning-icons-text/

Reply:
BTW, I think I have resolved the issue.  The new code I checked in works fine and I was able to remove the surrounding div.